internal/oidc/store.go

```diff
@@ -14,51 +14,30 @@ import (
 )
 
 // NewOpenIDConnectStore returns a new OpenIDConnectStore using the provided schema.OpenIDConnectConfiguration.
-func NewOpenIDConnectStore(configuration *schema.OpenIDConnectConfiguration) (store *OpenIDConnectStore) {
-	store = &OpenIDConnectStore{}
+func NewOpenIDConnectStore(configuration *schema.OpenIDConnectConfiguration) (store *OpenIDConnectStore, err error) {
+	store = &OpenIDConnectStore{
+		memory: &storage.MemoryStore{
+			IDSessions:             make(map[string]fosite.Requester),
```

Comment:
why instantiating differently compared to the others below?

Reply:
Oh I'm not sure why we did that, that's preexisting, let me change it and see if there was a  reason, but doubt it.